crowbar_framework/app/models/service_object.rb

```diff
@@ -1551,26 +1544,6 @@ def run_remote_chef_client(node, command, logfile_name)
     }
   end
 
-  def chef_daemon(action, node_list)
```

Comment:
wouldn't it be enough to just take out the caller method as i did in https://github.com/MaximilianMeister/barclamp-crowbar/commit/7b937bf64e14ca3c6b4eda169440b3ad7c661e31 as part of https://github.com/crowbar/barclamp-crowbar/pull/1346? maybe we could need this method later on.